gitops/openshift/rbac/template/imagepuller.env

```diff
@@ -0,0 +1,4 @@
+OC_NAMESPACE=17b24e
```

Comment:
I've created a secret for this: OPENSHIFT_SA_NAMESPACE - should we use this instead?